src/core/control/tools/TextEditor.cpp

```diff
@@ -319,14 +319,19 @@ auto TextEditor::onKeyPressEvent(GdkEventKey* event) -> bool {
             toggleBoldFace();
             return true;
         }
-        if (event->keyval == GDK_KEY_plus) {
+        if (event->keyval == GDK_KEY_plus || event->keyval == GDK_KEY_KP_Add) {
             increaseFontSize();
             return true;
         }
-        if (event->keyval == GDK_KEY_minus) {
+        if (event->keyval == GDK_KEY_minus || event->keyval == GDK_KEY_KP_Subtract) {
             decreaseFontSize();
             return true;
         }
+    } else if ((event->state & modifiers) == (GDK_CONTROL_MASK | GDK_SHIFT_MASK)) {
+        if (event->keyval == GDK_KEY_plus || event->keyval == GDK_KEY_KP_Add) {
```

Comment:
Could you add a comment as to why this case is special? Why `Ctrl`+`Shift`+`+` and not also `Ctrl`+`Shift`+`-`?
Is it related to some keyboard layouts having the + in a "Caps" position?

Reply:
I found a keyboard layout independent solution using consumed modifiers.